test/logflare/rules/rule_cache_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Logflare.Rules.CacheTest do
+  alias Logflare.Rules.Rule
+  use Logflare.DataCase
+
+  alias Logflare.Rules
+  alias Logflare.Sources
+
+  @subject Rules.Cache
+
+  setup do
+    Cachex.reset(@subject)
+    on_exit(fn -> Cachex.reset!(@subject) end)
```

Comment:
lets move cache resetting to DataCase and ConnCase, there is already some logic there